feat(biz): initial version (#517)
ethereum-optimism/community-hub: src/docs/biz/README.md

- Demo day, every two weeks.
  [Apply on our Discord](https://discord-gateway.optimism.io/) at the **#demoday** channel.
- For larger projects, we also do dedicated Twiter spaces.  

### Discovery

- Listing on our [website](https://www.optimism.io/apps/all).
  [Apply using our marketing request form](https://optimismpbc.typeform.com/op-marketing).
- Listing for your token on [our bridge](https://app.optimism.io/bridge).
  [Listing instructions here](https://github.com/ethereum-optimism/ethereum-optimism.github.io).
- [Quests](https://app.optimism.io/quests).
  [Apply using our marketing request form](https://optimismpbc.typeform.com/op-marketing).

## Get help (technical support)

- Developer support
  - [Fantastic developer docs](../developers)
  - Live support on [our Discord forum](https://discord-gateway.optimism.io/) at the **#dev-support** channel.
  - Office hours every two weeks, 2pm UTC, on Discord. 
- User support
  - [User help center](https://help.optimism.io/hc/en-us)
  - Community run user support [on our Discord](https://discord-gateway.optimism.io/) at the **#user-support** channel.

> ## Get funded

Reviewer: Cannot say this. Please change to get OP

> There are several ways in which Optimism can help with your funding.

Reviewer: Same here. 
```suggestion
There are several ways in which Optimism can give you OP.
```

- **Partnership grants**.
  Private process managed by The Optimism Foundation.
  Better for nuanced requirements.
  [Apply using our marketing request form](https://optimismpbc.typeform.com/op-marketing).

> - **Governance grants**.

Reviewer: Please put gov first

---

> title: Business Support for Optimistic Dapps

Reviewer: Update this
```suggestion
title: Support for Optimistic Dapps
```

- Listing for your token on [our bridge](https://app.optimism.io/bridge).
  [Listing instructions here](https://github.com/ethereum-optimism/ethereum-optimism.github.io).
- [Quests](https://app.optimism.io/quests).
  [Apply using our marketing request form](https://optimismpbc.typeform.com/op-marketing).

## Get help (technical support)

- Developer support
  - [Fantastic developer docs](../developers)
  - Live support on [our Discord forum](https://discord-gateway.optimism.io/) at the **#dev-support** channel.
  - Office hours every two weeks, 2pm UTC, on Discord. 
- User support
  - [User help center](https://help.optimism.io/hc/en-us)
  - Community run user support [on our Discord](https://discord-gateway.optimism.io/) at the **#user-support** channel.

## Get OP

There are several ways to receive OP tokens from the Optimism Foundation.

- **Governance grants**.
  Public process managed by our [token house](../governance/token-house.md).
  Typically milestone based, iterative grants.
  [Instructions how to apply here](../governance/proposals.md).

- **Partnership grants**.
  Private process managed by The Optimism Foundation.
  Better for nuanced requirements.

>   [Apply using our marketing request form](https://optimismpbc.typeform.com/op-marketing).

Reviewer: Please add a section called "Ecosystem Offerings" and add the Alchemy affiliate link (I think our link gives you more credits on a free tier or something?)
